tests/Unit/Integrations/Admin/Integrations_Page_Integration_Test.php

```diff
@@ -105,6 +105,9 @@ public function test_register_hooks() {
 	/**
 	 * Tests the registration of the hooks.
 	 *
+	 * @runInSeparateProcess
+	 * @preserveGlobalState disabled
```

Comment:
This is quite expensive. It is very much preferred to fix the source of the problem instead 😅